single_tech_samples/storage/sample1_data_lifecycle/README.md

```diff
@@ -0,0 +1,224 @@
+
+### 1.1. Scope
+
+The following items will be provisioned as a part of this sample setup:
+
+- Azure Storage Accounts.
+- Storage Containers.
+- Data lifecycle rules.
+
+Details about [how to use this sample](#2-how-to-use-this-sample) can be found in the later sections of this document.
+
+### 1.2. Use Case
+
+It is desirable to reduce the costs by optimizing the resource usage costs in cloud environments.
+
+Data lifecycle is one such feature on Azure using which you can optimize costs by moving your data to lower cost tiers when their access is less frequent. It also supports deleting obselete data to save costs.
+
+More details can be found in [this link](https://learn.microsoft.com/azure/storage/blobs/lifecycle-management-overview)
+
+We can define data lifecycle rules to move data from one tier to other. The current solution provides Infra As Code script written in Terraform, to provision these rules in Azure.
+
+Details about [how to run the pipeline](#24-deployment-validation-and-execution) can be found in the later sections of this document.
+
+### 1.3. Technologies used
+
+The following technologies are used to build this sample:
+
+- [Azure Storage](https://azure.microsoft.com/services/storage/data-lake-storage)
+- [Terraform](https://www.terraform.io)
+
+## 2. How to use this sample
+
+This section holds the information about usage instructions of this sample.
+
+### 2.1. Prerequisites
+
+The following are the prerequisites for deploying this sample:
+
+- [Github account](https://github.com)
+- [Azure Account](https://azure.microsoft.com/free/search/?&ef_id=Cj0KCQiAr8bwBRD4ARIsAHa4YyLdFKh7JC0jhbxhwPeNa8tmnhXciOHcYsgPfNB7DEFFGpNLTjdTPbwaAh8bEALw_wcB:G:s&OCID=AID2000051_SEM_O2ShDlJP&MarinID=O2ShDlJP_332092752199_azure%20account_e_c__63148277493_aud-390212648371:kwd-295861291340&lnkd=Google_Azure_Brand&dclid=CKjVuKOP7uYCFVapaAoddSkKcA)
+  - *Permissions needed*:  The ability to create and deploy to an Azure [resource group](https://docs.microsoft.com/azure/azure-resource-manager/management/overview), a [service principal](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals), and grant the [collaborator role](https://docs.microsoft.com/azure/role-based-access-control/overview) to the service principal over the resource group.
+
+  - Active subscription with the following [resource providers](https://docs.microsoft.com/azure/azure-resource-manager/management/azure-services-resource-providers) enabled:
+    - Microsoft.Storage
+
+#### 2.1.1 Software Prerequisites
+
+- [Azure CLI](https://docs.microsoft.com/cli/azure) installed on the local machine
+  - *Installation instructions* can be found [here](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [Terraform](https://www.terraform.io) installed on the local machine
+  - *Installation instructions* can be found [here](https://developer.hashicorp.com/terraform/downloads)
```

Comment:
Remove tilting "*" and add "." at the end.

Reply:
Done